src/lightning/pytorch/CHANGELOG.md

```diff
@@ -267,6 +267,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Removed the unused `lightning.pytorch.utilities.metrics.metrics_to_scalars` function ([#16681](https://github.com/Lightning-AI/lightning/pull/16681))
 
+- Removed legacy argparse utilities ([#16708](https://github.com/Lightning-AI/lightning/pull/16708))
```

Comment:
We can also remove the code in `src/lightning/pytorch/utilities/parsing.py`

Reply:
Done